libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.Bot.Builder.Adapters.Webex
     internal static class WebexHelper
     {
         /// <summary>
-        /// Creates a <see cref="Activity"/> using the body of a request.
+        /// Creates an <see cref="Activity"/> using the body of a request.
         /// </summary>
         /// <param name="payload">The payload obtained from the body of the request.</param>
         /// <param name="identity">The identity of the bot.</param>
```

Comment:
As per updated advice on other adapters, please throw an ArgumentNullException instead of returning null. Thanks.

Reply:
Done. Thank you!